lib/Models/Catalog/SdmxJson/SdmxJsonDataflowStratum.ts

```diff
@@ -601,6 +626,63 @@ export class SdmxJsonDataflowStratum extends LoadableStratum(
     ]);
   }
 
+  /** Get region TableColumn by searching catalogItem.tableColumns for region dimension
+   * NOTE: this is searching through catalogItem.tableColumns to find the completely resolved regionColumn
+   * This can only be used in computeds/fns outside of ColumnTraits - or you will get infinite recursion
+   */
+  @computed get resolvedRegionColumn() {
+    return this.catalogItem.tableColumns.find(
+      col =>
+        col.name ===
+        this.dimensionColumns.find(col => col.type === "region")?.name
```

Comment:
Is it  better to use `TableColumnType.region` here?

Reply:
Sorry I should use different local variables for the nested `.find()` calls. 

The second one is searching through `dimensionColumns` - which is `TableColumnTraits` - so it needs to search for the `"region"` string instead of `TableColumnType.region`